Fix Twitter, Together! README link

dfarrell07 opened the pull request:
We use devel as our main branch, fix Twitter, Together! README link
accordingly. Also use a normal URL instead of path for clarity.

Fixes: #1655
Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our developer guide: https://submariner.io/development/
2. Ensure you have added the appropriate tests for your PR: https://submariner.io/development/code-review/#test-new-functionality
3. Read the code review guide to ease the review process: https://submariner.io/development/code-review/
4. If the PR is unfinished, mark it as a draft: https://submariner.io/development/code-review/#mark-work-in-progress-prs-as-drafts
5. If you are using CI to debug, use your private fork: https://submariner.io/development/code-review/#use-private-forks-for-debugging-prs-by-running-ci
6. Add labels to the PR as appropriate.

This template is based on the K8s/K8s template:

https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md
-->

skitt reviewed (changes requested):
How about fixing the link instead? The working link is

    https://github.com/submariner-io/submariner/new/devel/?filename=tweets/%3Cyour-path%3E.tweet

dfarrell07 committed:
Fix Twitter, Together! README link

We use devel as our main branch, fix Twitter, Together! README link
accordingly. Also use a normal URL instead of path for clarity.

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>
